### PR TITLE
Adjust tests for sle 15 welcome screen

### PR DESCRIPTION
**Pull request by rwx788**
With new architecture we show license after product is selected. Hence,
it cannot be shown before this step. This adjusts flow for SLE15.

Note, this already will fix major part of failing tests, but not all.
[NEEDLES](https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/471)

[Verification run](http://10.160.66.147/tests/1997#)

### CODE REVIEW

**Comment by rwx788** [comment]
@okurz I expect that this should fix most of the tests, so we can merge now and I'll create a new PR for other tests I can find (e.g. multipath)

**Comment by okurz** [comment]
s/Adhust/Adjust/

File: lib/y2logsstep.pm [hunk 1]
```diff
@@ -218,6 +228,7 @@ sub post_fail_hook {
     assert_script_run 'df > /tmp/df.txt';
     upload_logs '/tmp/df.txt';
 
+
```
Comment (okurz) [inline lib/y2logsstep.pm]: unrelated

**Comment by okurz** [comment]
I suggest you call something like `tools/tidy && git commit --amend -C HEAD -a && git pr-last` to incorporated tidied changes and not need to revisit the tidy step all the time

**Comment by okurz** [comment]
needles merged
